### DISCUSSION TITLE
dynamic bias

### DISCUSSION THREAD

**Discussion by AliRamshini** [rodralez/NaveGo#65] [Q&A] [answered]
Hi Dr
I Checked the True Dynamic bias and estimated Dynamic bias from Kalman Filter but I think Estimated Dynamic bias don't Tracking The True Dynamic bias ???
Thanks a lot for your best code . . .

**Comment by rodralez** [comment, marked as answer]
Hi Ali,

I understand that you are talking about the NaveGo's example using synthetic data.

As you can see in function noise_b_dyn(), I have implemented two ways to model de dynamic bias, Aggarwal and Nassar methods. By default, the Nassar method is used. You may try the Aggarwal method and see if the EKF offers a better performance.

Best regards.
